src/wp-consent-api.js

```diff
@@ -63,7 +63,7 @@ function consent_api_set_cookie(name, value, path = '/') {
 
     if (window.location.protocol !== "https:") secure = '';
 
-    document.cookie = name + "=" + value + secure + expires + ";path=" + path;
+    document.cookie = name + "=" + value + secure + expires + ";path=" + ( path !== undefined ? path : '/' );
```

Comment:
I think `( path || '/' )` would also work.

Reply:
Yeah, I just opted to be explicitly checking if the value is provided or not, regardless of the fact that any falsey value wouldn't work.